openmc/material.py

```diff
@@ -908,22 +899,49 @@ def get_nuclide_atom_densities(self):
 
         return nuclides
 
-    def get_nuclide_activity(self):
-        """Return activity in [Bq] for each nuclide in the material
+    def get_activity(self, normalization: str = 'total', by_nuclide: bool = False):
+        """Returns the activity of the material or for each nuclide in the
+        material in units of [Bq], [Bq/g] or [Bq/cc].
 
         .. versionadded:: 0.13.1
 
+        Parameters
+        ----------
+        normalization : {'total', 'mass', 'volume'}
+            Specifies the type of activity to return, 'total' will return the
+            material activity in [Bq], 'mass' returns the materials specific
+            activity in [Bq/g] and 'volume' returns the activity in [Bq/cc].
+        by_nuclide : bool
+            Specifies if the activity should be returned for the material as a
+            whole or per nuclide.
+
         Returns
         -------
-        dict
-            Dictionary whose keys are nuclide names and values are activity in
-            [Bq].
+
+        Union[dict, float]
+            If by_nuclide is True then a dictionary whose keys are nuclide 
+            names and values are activity is returned. Otherwise the activity
+            of the material is returned as a float.
         """
+
+        cv.check_value('normalization', normalization, {'total', 'mass', 'volume'})
+        cv.check_type('by_nuclide', by_nuclide, bool)
+        
+        if normalization == 'total':
+            multiplier = self.volume
+        else:
+            multiplier = 1 if normalization == 'volume' else 1.0 / self.get_mass_density()
```

Comment:
This seems a little more clear to me. Could add a `ValueError` at the end, but `normalization` is checked above so I feel okay about this.

```suggestion
        if normalization == 'total':
            multiplier = self.volume
        elif normalization == 'volume':
            multiplier = 1
        elif normalization == 'mass':
            multiplier = 1.0 / self.get_mass_density()
```

Reply:
works for me